### ISSUE TITLE
"dcl link" command with no need for Metamask

### ISSUE THREAD

**Issue by abarmat** [decentraland/cli#41]
We can make the **dcl link** command to use a Geth node to do all the validations about ownership of parcels and even sign the transaction to update the parcel and avoid having a web interface for that.

To do this we can use: an Infura node, a Decentraland hosted node, or an Ethereum node hosted by the user.

**Comment by HAZARDU5** [comment]
This would be nice, especially for those of us running the cli inside a headless Linux VM. Currently we'd need to install a web browser with Metamask plugin inside VM to do the linking workflow.

**Comment by bnolan** [comment]
I think being able to sign the request on the command line is a good idea, but what about if instead of requesting the users passphrase, we generate the `geth` code to sign the request, so that the user can copy and paste the request into their command line wallet and generate the request? cc @eordano  @abarmat

**Comment by abarmat** [comment]
It will work, but I think it's not very convenient to do it manually for the user. 

1) I was thinking pasting a private key => encrypt it with a passphrase during setup.
2) When we need to send a TX we sign and broadcast it to the network.

**Comment by menduz** [comment]
How many NPM packets does the CLI install? 

~1.5k?

If I can take control of any NPM token of any of those packets due a problem with TravisCI configuration, I can gain access to the computer that is executing this code and replace the console to take control of the pasted code.

IMHO we should handle the private keys in a completely different execution context, in this case Metamask

**Comment by bnolan** [comment]
I agree with @menduz, I don't think we should be handling private keys, we should provide an easy way for the user to sign the transaction using geth, metamask, a ledger or trezor by copy and pasting the hex code. Managing the keys ourselves is ripe with dangers.

**Comment by HAZARDU5** [comment]
Yeah I don't know if I'd be comfortable entering my private keys into the CLI tooling.

**Comment by abarmat** [comment]
I agree with @menduz that it's dangerous until we have a tool to check the full tree of npm dependencies, @ntrippar is on it.

I think for convenience it is quite easy to use Metamask, for the advanced user it might be interesting to have the tx prepared to sign but I don't see it as a priority feature.
